controllers/positions.js

```diff
@@ -0,0 +1,69 @@
+const fs = require('fs')
+const Positions = require('../data/positions.json')
+
+/** Description. takes params: idClient, jobDescription, vacancy, professionalProfiles and status. ID is generated automatically*/
+const createPosition = (req, res) => {
+  const newPosition =
+    {
+      id: new Date().getTime().toString(),
+      idClient: req.query.idClient,
+      jobDescription: req.query.jobDescription,
+      vacancy: req.query.vacancy,
+      professionalProfiles: req.query.professionalProfiles,
+      status: req.query.status
+    }
+    Positions.push(newPosition)
+    fs.writeFile('./data/positions.json', JSON.stringify(Positions), {}, err => {
+      if (err) throw err
+    })
+    res.send('created position')
+}
+
+/** Description. takes params: id (this is the id you want to update) , idClient, jobDescription, vacancy, professionalProfiles and status.*/
+const updatePosition = (req, res) => {
+  const positionIdToUpdate = req.query.id
+  for (const position in Positions) {
+    if(Positions[position].id == positionIdToUpdate){
+      Positions[position].idClient = req.query.idClient,
+      Positions[position].jobDescription = req.query.jobDescription,
+      Positions[position].vacancy = req.query.vacancy,
+      Positions[position].professionalProfiles = req.query.professionalProfiles,
+      Positions[position].status = req.query.status
+      res.send(`updated position id:${positionIdToUpdate}`)
+      break //there should be just 1 position to update, each is unique
+    }
+  }
+  fs.writeFile('./data/positions.json', JSON.stringify(Positions), {}, err => {
```

Comment:
Remember every endpoint should have a response. Put a `req.send` into the `fs.writeFile` callback for both cases (success and error)